linux_os/guide/system/accounts/accounts-session/accounts_logon_fail_delay/rule.yml

```diff
@@ -11,7 +11,7 @@ rationale: |-
     Increasing the time between a failed authentication attempt and re-prompting to
     enter credentials helps to slow a single-threaded brute force attack.
 
-severity: medium
+severity: low
```

Comment:
I agree with the original severity over this change.

Reply:
Ok. reverting